src/Symfony/Component/Translation/Tests/Translatable/MoneyTranslatableTest.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Translation\Tests\Translatable;
+
+use Money\Money;
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Translation\Translatable\MoneyTranslatable;
+use Symfony\Contracts\Translation\TranslatorInterface;
+
+class MoneyTranslatableTest extends TestCase
+{
+    /**
+     * @dataProvider getValues()
+     */
+    public function testTrans(string $expected, MoneyTranslatable $parameter, string $locale)
+    {
+        $translator = $this->createMock(TranslatorInterface::class);
+        // Non-breakable spaces are added differently depending the PHP version
```

Comment:
should we normalize it by default?

Reply:
I don't know 🤔 
* On one side, this is a good idea to always have the same output: so the output should use only regular spaces or only spaces are non-breakable
* On the other, I'm no expert of these NBSP and there must be some good reasons if they change from one PHP version to the other

And I don't have the right setup to run extensive tests on all PHP major/minor/patches versions.

I've added this line because I ran into issues with Symfony CI multiple pipelines.

What do you recommend?

Reply:
I think we should preseve _and assert_ the string INTL gives us.

The NBSP in that sense serves a purpose/practice money values shouldnt break, separating the currency from the decimal. But i'm not sure how it differs accross locales, hence i think it's best to preserve it as-is.

Reply:
@ro0NL Yes you're right but please note that the normalization is only done in the test to prevent it from breaking should a new PHP version introduces a change.

I think testing the right "style" used by DecimalMoneyFormatter is out of scope here.

I've added a comment in the test to explain why I'm doing it.